packages/block-library/src/cover/edit/index.js

```diff
@@ -70,7 +70,7 @@ function getInnerBlocksTemplate( attributes ) {
  */
 const isTemporaryMedia = ( id, url ) => ! id && isBlobURL( url );
 
-function CoverEdit( {
+export function CoverEdit( {
```

Comment:
Any reason why we can't use the default export?

Reply:
We actually don't need that export anymore - that was from some of my earlier experimentation - the default export was already used on the export with the `withColors` wrapper. Have reverted this now.